bean/config/.env.example

```diff
@@ -10,6 +10,7 @@ DB_PORT=5432
 DB_USERNAME=postgres
 DB_PASSWORD=postgres
 DB_SSL_MODE=disable
+DB_VERSION_TABLE=public.bean_schema_version
```

Comment:
Not a fan of this

Let's rename it to `DB_SCHEMA` 
And use `bean_schema_version` as the default schema name

Doesn't matter if it's "too verbose"